src/Components/CodeLensHelpers.fs

```diff
@@ -4,13 +4,10 @@ open System
 open Fable.Core.JsInterop
 open Fable.Import.VSCode
 open Fable.Import.VSCode.Vscode
-open global.Node
-
-module node = Node.Api
```

Comment:
these kinds of module node declarations were removed globally because of F# language warnings about partially-resolved namespace names.